### PR TITLE
Fix avatar size, set sidebar refresh 10s

### PR DESCRIPTION
**Pull request by aumu1031**

### CODE REVIEW

**Comment by aumu1031** [comment]
Included change to default (tier 0) forbidden seal avatar size limit, seems like it is supposed to be 150x150?

File: ui_components/src/sidebar/Sidebar.js [hunk 1]
```diff
@@ -131,8 +131,8 @@ function Sidebar({ links, logoutTimer, navigationAPIData, userAPIData }) {
         return (
             <>
                 <div className="sb_avatar_container">
-                    <div className="sb_avatar_wrapper">
-                        <img className="sb_avatar_img" src={playerData.avatar_link}/>
+                    <div className={playerData.avatar_size < 200 ? "sb_avatar_wrapper_default" : "sb_avatar_wrapper_large"}>
+                        <img className={playerData.avatar_size < 200 ? "sb_avatar_img_default" : "sb_avatar_img_large"} src={playerData.avatar_link}/>
```
Comment (levimeahan) [inline ui_components/src/sidebar/Sidebar.js:135]: The only difference between default and large is the size - For maximum flexibility here, how about leaving off the width from the regular stylesheet and declaring inline using `playerData.avatar_size`?

```suggestion
                    <div className="sb_avatar_wrapper">
                        <img className="sb_avatar_img" style={{ maxWidth: playerData.avatar_size, maxHeight: playerData.avatar_size}} src={playerData.avatar_link}/>
```

File: ui_components/src/sidebar/Sidebar.css [hunk 2]
```diff
@@ -93,7 +93,17 @@
...
-.sb_avatar_wrapper {
+.sb_avatar_wrapper_default {
+    clip-path: circle(50% at 50% 50%);
+    background: var(--avatar-frame-border_gradient);
+    height: 137px;
+    width: 137px;
+    position: relative;
+    left: 50%;
+    transform: translateX(-50%);
+}
```
Comment (levimeahan) [inline ui_components/src/sidebar/Sidebar.css:104]:
```suggestion
.sb_avatar_wrapper {
    clip-path: circle(50% at 50% 50%);
    background: var(--avatar-frame-border_gradient);
    padding: 6px;
    position: relative;
    left: 50%;
    transform: translateX(-50%);
}
```

File: ui_components/src/sidebar/Sidebar.css [hunk 3]
```diff
@@ -103,7 +113,16 @@
     transform: translateX(-50%);
 }
 
-.sb_avatar_img {
+.sb_avatar_img_default {
+    position: relative;
+    top: 6px;
+    left: 6px;
+    height: 125px;
+    width: 125px;
```
Comment (levimeahan) [inline ui_components/src/sidebar/Sidebar.css:121]:
```suggestion
```

**Review by levimeahan — APPROVED** [review]

**Pull request merged**